halo2_proofs/src/poly/domain.rs

```diff
@@ -474,17 +515,35 @@ impl<F: WithSmallOrderMulGroup<3>> EvaluationDomain<F> {
             omega: &self.omega,
         }
     }
+
+    /// Get the private field `n`
+    pub fn get_n(&self) -> u64 { self.n }
+
+    /// Get the private `fft_data`
+    pub fn get_fft_data(&self, l: usize) -> &FFTData<F> {
+            if l == self.fft_data.get_n() {
+                &self.fft_data
+            } else {
+                &self.extended_fft_data
+            }
+        }
```

Comment:
Strange indentation?

Reply:
Indeed.  It seems like my "Format Selection" feature in Vscode is broken.  Sorry for the inconvenience.